api/test/pact/pact-tests/get-caseworker-by-id.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { Pact } from '@pact-foundation/pact'
+import { assert } from 'chai'
+import * as getPort from 'get-port'
+import * as path from 'path'
+import { EnhancedRequest } from '../../../lib/models'
+import { handleCaseWorkerDetails } from '../../../workAllocation/caseWorkerService'
+
+describe("Work Allocation for location and service Caseworker API", () => {
+
+    let MOCK_SERVER_PORT
+    let workallocationUrl
+    let provider
+    const mockResponse = {
+          "firstName": "firstName",
```

Comment:
Double quotes on the name, not required here.

Reply:
Done